### DISCUSSION TITLE
BPF Loader Pull Request

### DISCUSSION THREAD

**Discussion by ijuedt** [bmresearch/Solnet#390] [General]
Is there any idea when the BPF Loader pull request could go live? Seems an important feature to me, and I'd be using it right away. I'd like to be giving a hand myself, but I am not deeply enough intimate with the inner workings of Solana to be reliable or efficient here.

br
Ingbert

**Comment by Neo-vortex** [comment]
Actually I had some difficulties lately, will try to finish unit test and ask for the merge again

**Reply by ijuedt** [reply]
Thanks for the quick reply, and good luck!
